runelite-client/src/main/java/net/runelite/client/plugins/smelting/SmeltingPlugin.java

```diff
@@ -103,6 +103,14 @@ public void onChatMessage(ChatMessage event)
 			}
 			session.increaseBarsSmelted();
 		}
+		else if (event.getMessage().startsWith("The Varrock platebody enabled you to smelt your next ore"))
```

Comment:
You can just add this to the if above
```java
if (event.getMessage().startsWith("You retrieve a bar of") || event.getMessage().startsWith("The Varrock platebody enabled you to smelt your next ore"))
```
as the code is the same.

Reply:
Works for me!  Will get an update out.

Reply:
Done :-)